File: RetiocusWebUtils/UtilsEntities/CWLMQueryPackage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UtilsEntities
{
    public class CWLMQueryPackage
    {
        public List<ChatDetailWLM> listadoChats { get; }

        public CWLMQueryPackage(List<ChatDetailWLM> listadoChats)
        {
            this.listadoChats = listadoChats;
        }
    }
}

File: RetiocusApp/app/src/main/java/com/jruiz/retiocusapp/signalr/models/CWLMQueryPackage.java
package com.jruiz.retiocusapp.signalr.models;

import java.util.List;

public class CWLMQueryPackage {
    private List<ChatDetailWLM> listadoChats;

    public CWLMQueryPackage(List<ChatDetailWLM> listadoChats) {
        this.listadoChats = listadoChats;
    }

    public List<ChatDetailWLM> getListadoChats() {
        return listadoChats;
    }

    public void setListadoChats(List<ChatDetailWLM> listadoChats) {
        this.listadoChats = listadoChats;
    }
}
